lib/src/rules/prefer_collection_literals.dart

```diff
@@ -32,6 +32,28 @@ var ids = <int>{};
 var coordinates = <int,int>{};
 ```
 
+**EXCEPTIONS:**
+
+There are cases with LinkedHasSet or LinkedHashMap where a literal constructor
```

Comment:
LinkedHasSet => LinkedHashSet

Also, consider formatting these as code? As in \`LinkedHashSet\`?

Reply:
Sure thanks